packages/frontend/src/pages/donate/getProps.ts

```diff
@@ -49,6 +49,10 @@ export async function getProps(
             name: 'zkSync 1.0',
             linkURL: `https://zkscan.io/explorer/accounts/${address}`,
           },
+          {
+            name: 'Gitcoin',
+            linkURL: 'https://gitcoin.co/grants/3857/l2beat',
```

Comment:
This is unnecessary, because during gitcoin rounds we show a big button and otherwise direct donations are nicer